extras/jenkins/Minikube/Jenkinsfile

```diff
@@ -185,6 +180,7 @@ _EOF_
             $WORKSPACE/pulumi/python/venv/bin/pulumi config set prometheus:adminpass "password" -C pulumi/python/config -s marajenk${BUILD_NUMBER}
             $WORKSPACE/pulumi/python/venv/bin/pulumi config set prometheus:helm_timeout "600" -C pulumi/python/config -s marajenk${BUILD_NUMBER}
             $WORKSPACE/pulumi/python/venv/bin/pulumi config set prometheus:helm_timeout "600" -C pulumi/python/config -s marajenk${BUILD_NUMBER}
+            '''
```

Comment:
oh noooo.  Glad this is fixed 🐳

Reply:
That was a bit annoying, and oddly hard to find. It was also weird that the Jenkinsfile linter did not seem to pick it up.